db/migrate/20190408145933_add_not_null_constraints.rb

```diff
@@ -0,0 +1,13 @@
+class AddNotNullConstraints < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+  def up
+    execute "ALTER TABLE assignments ADD CONSTRAINT assignments_force_check CHECK (force IS NOT NULL) NOT VALID;"
+    execute "ALTER TABLE previous_assignments ADD CONSTRAINT previous_assignments_force_check CHECK (force IS NOT NULL) NOT VALID;"
+    execute "ALTER TABLE bulk_assignments ADD CONSTRAINT bulk_assignments_force_check CHECK (force IS NOT NULL) NOT VALID;"
```

Comment:
These constraints will not be represented in the schema.rb because they're check constraints, not usual column not null declarations. That's sort of the crux of why I'm doing weird stuff here.